Doc/Tutorial/TutorialBasicMonomorphic.lhs

```diff
@@ -149,9 +143,6 @@ them.
 > instance Default Unpackspec BirthdayField BirthdayField where
 >   def = birthdayFieldDef
 
-Naturally this is all derivable using `Generic` or Template Haskell,
-but no one's bothered to implement that yet.  Would you like to?
-
```

Comment:
This comment is specifically about deriving for *monomorphic* record types, so the comment is still true.

Reply:
Ok, change reverted.